parsing/cLexer.mli

```diff
@@ -8,8 +8,32 @@
 (*         *     (see LICENSE file for the text of the license)         *)
 (************************************************************************)
 
+(** When one registers a keyword he can declare it starts a quotation.
```

Comment:
Try to avoid the non-gender-neutral "he". You can simply replace it by "they" (plural acting as gender-neutral singular). If you don't like this, then "she" is still preferable to "he" to acknowledge that not all developers are male.

Reply:
done